frontend/appflowy_flutter/lib/plugins/document/application/editor_transaction_adapter.dart

```diff
@@ -183,17 +183,15 @@ extension on InsertOperation {
       final parentId = node.parent?.id ??
           editorState.getNodeAtPath(currentPath.parent)?.id ??
           '';
-      var prevId = previousNode?.id;
+
+      String prevId = '';
       // if the node is the first child of the parent, then its prevId should be empty.
       final isFirstChild = currentPath.previous.equals(currentPath);
+
       if (!isFirstChild) {
-        prevId ??= editorState.getNodeAtPath(currentPath.previous)?.id ?? '';
-      }
-      prevId ??= '';
-      assert(parentId.isNotEmpty);
-      if (isFirstChild) {
-        prevId = '';
-      } else {
+        prevId = previousNode?.id ??
+            editorState.getNodeAtPath(currentPath.previous)?.id ??
+            '';
```

Comment:
@LucasXu0 I rewrote this part of the logic. I believe it's the same but can you double check just to be safe?

Reply:
I think this code was deleted accidentally.

```dart
      assert(parentId.isNotEmpty);
```

Reply:
Ok I've added this back